contrib/linux/org.dosbox-staging.DOSBoxStaging.metainfo.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright (C) 2020-2024  The DOSBox Staging Team -->
 <component type="desktop">
-  <id>io.github.dosbox-staging</id>
+  <id>org.dosbox-staging.DOSBoxStaging</id>
```

Comment:
Why not just `org.dosbox-staging`? Rest assured, we won't add a second project, so `.org.dosbox-staging.dosbox-staging` seems very redudant and a bit ugly.

Reply:
That will not be accepted for e.g., flatpak. it is <domain>.<app>, even if it is redundant.

Reply:
Ah well, ok then.